sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/AbstractDAIAdapter.java

```diff
@@ -116,9 +127,9 @@ public AbstractDAIAdapter<? extends SclElementAdapter> update(Map<Long, String>
      * @throws ScdException throws when DAI for which SGroup should be updated is not updatable
      */
     public void update(Long sGroup, String val) throws ScdException {
-        if (currentElem.isSetValImport() && !currentElem.isValImport()) {
+        if (!isDOModDAstVal() && currentElem.isSetValImport() && !currentElem.isValImport()) {
```

Comment:
I'm wondering if it's possible to remove [AbstractDAIAdapter.isValImport](https://github.com/com-pas/compas-sct/blob/239-exclude-doiModstval-for-dais-updatability/sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/AbstractDAIAdapter.java#L98)  method because it's only used in tests and can cause confusion in the future.

Reply:
Removed